index.html

```diff
@@ -12,29 +12,78 @@
 
         <link href="@css/main_style.css" rel="stylesheet" type="text/css">
         
+        <link href="@css/standard_colors_classes.css" rel="stylesheet" type="text/css">
+        
+        <script src="https://kit.fontawesome.com/e44bf54afd.js" crossorigin="anonymous"></script>
+        
     </head>
 
-    <body>
+    <body class="w-375px">
+
         <!-- Page HEADER block -->
-        <header class="flex-box header">
+        <!-- This block works as the toolbar container -->
+        <header class="flex-box horizontal header white-bg blue-fg w-375px">
 
             <!-- Logo -->
-            <div class="flex-box logo">
-                <a href="index.html"><img class="logo_image" src="@images/ajg_logo.svg" alt="Logotype image" /></a>
+            <div class="flex-box horizontal logo">
+                <a href="index.html" target="_self" rel=""><img class="logo_image" src="@images/ajg_logo.svg" alt="Logotype image" /></a>
             </div>
 
             <!-- Page NAV (Menu) block -->
-            <nav class="flex-box nav-bar">
+            <nav class="flex-box horizontal nav-bar black-fg white-bg">
                 <ul>
-                  <li>About me</li>
+                  <li>
+                      <a href="index.html" target="_self" rel="">
+                          <i class="fa-solid fa-bars fa-1x blue-fg white-bg"></i>
+                      </a>
+                  </li>
                 </ul>
             </nav>
 
         </header>
-
+ 
 
         <!-- Page main body block -->
-        <div class="flex-box main-body">
+        <div class="flex-box vertical main-body">
+
+            <!-- First section -->
+            <section class="flex-box vertical flex-center transparent-bg black-fg" id="section_1" >
+
+                <h1>I am Marianne<br/>Glad to see<br/>you!</h1>
+
+                <p>I am a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+
+                <div class="flex-box vertical flex-center">
+                    <h3 class="blue-fg">LET'S CONNECT</h3>
+                    <ul class="flex-box horizontal flex-start">
+                        <li class="no-bullet">
```

Comment:
- [ ] great job. Please for the main title and text the color font color is #172B4D and for the let's connect button, the color is #4053FC
- [ ] please kindly set the width to 100% to make it cover the whole page and be responsive as well.(375-768px). you can set the backround-size to cover, so it covers the entire section.
- [ ] please make the main header bolder and bigger like in the original design.
- [ ] as seen in the Figma version, the color of the headline section background is white
yours
![Screenshot (1216)_LI](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/65298048/3ef68743-9a99-4f86-8a64-171488564ba0)
Figma design
![Screenshot (1215)](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/65298048/7df65f0d-8e7b-4c4f-81da-2e536a6079af)